tools/test_configs/EthernetInterface.json

```diff
@@ -7,6 +7,10 @@
         "echo-server-port" : {
             "help" : "Port of echo server",
             "value" : "7"
+        },
+        "echo-server-discard-port" : {
+            "help" : "Discard port of echo server",
+            "value" : "9"
```

Comment:
Hi
I am using a local test network.
Is this new setting need some new configuration command to my server ?
Thx

Reply:
This port number is the same as what was used earlier. The only difference is that instead of it being hardcoded now it can be configured.

Reply:
Hi,
No, there is no need to change the network/server configuration.
The tests were so far using the same port number, but it was coded explicitly, like so: 
`tcp_addr.set_port(9);`
Now it is defined in the config file, to avoid magic numbers, like so:
`tcp_addr.set_port(MBED_CONF_APP_ECHO_SERVER_DISCARD_PORT);`